code/game/area/areas.dm

```diff
@@ -288,11 +288,16 @@
 /area/space/powered(chan) //Nope.avi
 	return 0
 
-// called when power status changes
-
+/**
+  * Called when the area power status changes
+  *
+  * Updates the area icon and calls power change on all machinees in the area
+  * Updates the area icon, calls power change on all machinees in the area, and sends the `COMSIG_AREA_POWER_CHANGE` signal.
```

Comment:
Nitpick but the first line here can be removed, as its redundant. Also in the second line, there is a typo: "machinees"